docs/src/enterprise/getstarted/migrate-from-oss.md

```diff
@@ -7,13 +7,14 @@ description: How to migrate from lakeFS OSS to lakeFS Enterprise
 
 To migrate from lakeFS Open Source to lakeFS Enterprise, follow the steps below:
 
-1. Make sure you have the Fluffy Docker token, if not [contact us](https://lakefs.io/contact-sales/) to gain access to Fluffy. You will be granted with a token that enables downloading *dockerhub/fluffy* from [Docker Hub](https://hub.docker.com/u/treeverse).
-1. Update lakeFS docker image to enterprise. Use `treeverse/lakefs-enterprise` instead of `treeverse/lakefs`. The image can be pulled using the same token as Fluffy.
+1. Make sure you have the lakeFS Enterprise Docker token. if not, [contact us](https://lakefs.io/contact-sales/) to gain access to lakeFS Enterprise. You will be granted a token that enables downloading *dockerhub/lakeFS-Enterprise* from [Docker Hub](https://hub.docker.com/u/treeverse).
+1. Update the lakeFS Docker image to the enterprise version. Replace `treeverse/lakefs` with `treeverse/lakefs-enterprise` in your configuration. The enterprise image can be pulled using your lakeFS Enterprise token.
 1. Sanity Test (Optional): Install a new test lakeFS Enterprise before moving your current production setup. Test the setup > login > Create repository etc. Once everything seems to work, delete and cleanup the test setup and we will move to the migration process.
 1. Follow lakeFS [Enterprise installation guide][lakefs-enterprise-install]
     1. Make sure that you meet the [prerequisites][lakefs-enterprise-install-prerequisites]
```

Comment:
Remove. pre-requisites are already inside and this is irrelevant, only for people with k8s.